edenia/antelope-tools
# Feat/replace login with connect wallet

###  Replace login button

### What does this PR do?

- Resolve #1168 
- For the mobile design, the language selector has been moved to the sidebar.
- Keep the user on the same page when the logout button is pressed.

### Steps to test

1. Run the project locally
2. Go to webapp
3. See the button
4. Connect your wallet
5. Check your account name 
6. Logout


### Screenshots

Desktop version
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/6e84ad47-13a8-4d04-8cd6-40c096534244)

Mobile version
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d4a94ed9-89ed-4300-9cf2-c93a4fdb28a4)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/018d2ce1-db38-4b5a-b826-939238ad3769)

Change language in the sidebar for mobile version
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/9030f554-3e12-44cb-b8a9-a6082a450023)

Message when trying to execute an action before connecting the wallet
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/245a3898-23f8-4f69-a9e3-8aefe799b762)
